src/light.c

```diff
@@ -913,6 +932,51 @@ bool light_cmd_sub_brightness(light_context_t *ctx)
     return true;
 }
 
+bool light_cmd_mul_brightness(light_context_t *ctx)
+{
+    light_device_target_t *target = ctx->run_params.device_target;
+    if(target == NULL)
+    {
+        LIGHT_ERR("didn't have a valid target, programmer mistake");
+        return false;
+    }
+
+    uint64_t value = 0;
+    if(!target->get_value(target, &value))
+    {
+        LIGHT_ERR("failed to read from target");
+        return false;
+    }
+
+    uint64_t max_value = 0;
+    if(!target->get_max_value(target, &max_value))
+    {
+        LIGHT_ERR("failed to read from target");
+        return false;
+    }
+
+    value *= ctx->run_params.float_value;
```

Comment:
If value is zero and it is multiplied by >1... it will still be zero. Additionally, if value is some low number and it is multiplied by a float value, it could remain the same low value and the command will have no effect. This makes the functionality not very suitable for brightness up/down keyboard shortcuts, as the brightness could get stuck at a certain value. Perhaps check if float_value is >1 and force an increment for value in those cases (likewise for decrease), even if the multiplication operation would result in the value remaining the same.

Reply:
I would suggest the following:
```suggestion
    uint64_t old_value = value;
    value *= ctx->run_params.float_value;
    
    // Check that we actually de/increase value
    if( value == old_value ) {
        if( ctx->runs_params.float_value > 1 ) {
            value++;
        }
        if( ctx->runs_params.float_value < 1 && value > 0 ) {
            value--;
        }
    }
```
This should ensure that value is de/increased if desired by the user. Also avoid underflow if `value = 0`. Overflow should not be an issue.

Reply:
@TheJakeSchmidt The patch by @gerw looks OK. You can commit it to your pull-request directly by using the web-interface so we can merge this PR.

Reply:
Done, thanks @gerw!